components/switch/themes/spectrum.css

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+
+@container (--system: spectrum) {
+    .spectrum-Switch--emphasized {
+        --spectrum-switch-background-color-selected: var(--spectrum-accent-color-900);
+    }
```

Comment:
Interesting thing about this: `accent-color-900` holds both of the values for Spectrum and Express, and because of this, we do not (and should not) define this here. Let's take this whole selector and bring it over to the `index.css` file.

Just a reminder/helpful tip about when to use these `themes` files: we only want to define things in here that will change between Spectrum and Express. Also, if it helps to keep this in mind: any custom properties defined in Spectrum must also be defined in Express, so if you know that something like `--spectrum-bernhards-fancy-background` will be blue in Spectrum, but indigo in Express, you'll add `--spectrum-bernhards-fancy-background` in BOTH `spectrum.css` and `express.css` and the value (blue or indigo) will change between the two. Make sense?

Reply:
so I moved this to `index.css` and it losses the accent color. 
It does change between Spectrum and Express (blue vs. purple). 
<img width="564" alt="Screen Shot 2022-08-24 at 15 14 36" src="https://user-images.githubusercontent.com/52184321/186525629-e0e7073a-f1f4-488d-a982-e39b41fcbb6b.png">
<img width="465" alt="Screen Shot 2022-08-24 at 15 14 13" src="https://user-images.githubusercontent.com/52184321/186525636-694a3752-94ec-41e7-ab7c-629af13975ba.png">

Reply:
Ok.

I still think that it belongs within `index.css`, it's just that we need selectors defined in there to accommodate the styles for `--emphasized`.

Specifically, I think you'll want to have some code similar to this:

```css
.spectrum-Switch--emphasized {
  /* selected, emphasized */
  &:checked + .spectrum-Switch-switch {
    background-color: var(--highcontrast-switch-background-color-emphasized, var(--mod-switch-background-color-emphasized, var(--spectrum-switch-background-color-emphasized)));
    
  ...
  } 
}
```

I say this because we also need to handle the `hover`, `down`, and `keyboard focus` colors for "selected emphasized".

Does that make sense?